hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/StoreScanner.java

```diff
@@ -1086,5 +1120,4 @@ public long getEstimatedNumberOfKvsScanned() {
   public Cell getNextIndexedKey() {
     return this.heap.getNextIndexedKey();
   }
-}
-
+}
```

Comment:
nit, an empty line at the end of file

Reply:
I fixed nit a9443952ada0a3022f0c7f083fe9752c1d83f75a